apps/frontend/src/app/(app)/space/this-week/page.tsx

```diff
@@ -1,5 +1,137 @@
+"use client"
```

Comment:
Rather than making the entire page to render on the client side you can create separate components and wherever a user interaction is required, you can make that component as 'use client'.

This way static data will be able to render on the server side rather than on client side which will be better for performance.